### DISCUSSION TITLE
Autel MaxiCharge EVSE. What can I do to help make this work?

### DISCUSSION THREAD

**Discussion by Syracuse3DPrinting** [lbbrhzn/ocpp#669] [Ideas]
Hi all,

I've got an Autel Maxicharge EVSE set up using the OCPP integration per the instructions. The basics are working and I can get some info and charge control shows up but the EVSE has a couple of errors in the integration and becomes unavailable frequently. Wondering if there's anything I can do/share that would help to getting this EVSE added to the list of compliant hardware?

I'm not a programmer so it's nothing I'll be able to fully troubleshoot on my own, but if there's some data I can provide that would help someone who can I'm more than happy to help. 

Since the last upgrade the error notification I get is:

Warning: charger reported Rejected while setting MeterValuesSampledData=Energy.Active.Import.Register

### Device info
MaxiChargerAC
by Autel
Firmware: CAQYAPAQcA

A glance at my logs with no interaction:

> charger.Error.Code became unavailable
> 4:41:09 PM - 7 minutes ago
> 
> charger.Heartbeat became unavailable
> 4:41:09 PM - 7 minutes ago
> 
> charger.Status.Connector became unavailable
> 4:41:09 PM - 7 minutes ago
> 
> charger.Status became unavailable
> 4:41:09 PM - 7 minutes ago
> 
> charger.Transaction.Id became unavailable
> 4:41:09 PM - 7 minutes ago
> 
> charger.Availability turned on
> 4:39:06 PM - 9 minutes ago
> 
> charger.Charge_Control turned off
> 4:39:06 PM - 9 minutes ago
> 
> charger.Timestamp.Config.Response changed to October 31, 2022 at 12:17 AM
> 4:39:06 PM - 9 minutes ago
> 
> charger.Timestamp.Data.Transfer became unavailable
> 4:39:06 PM - 9 minutes ago
> 
> charger.Timestamp.Data.Response became unavailable
> 4:39:06 PM - 9 minutes ago
> 
> charger.Connectors changed to 1
> 4:39:06 PM - 9 minutes ago
> 
> charger.Features changed to Profiles.AUTH|REM|SMART|FW|CORE
> 4:39:06 PM - 9 minutes ago
> 
> charger.Version.Firmware changed to REDACTED
> 4:39:06 PM - 9 minutes ago
> 
> charger.Serial became unavailable
> 4:39:06 PM - 9 minutes ago
> 
> charger.Vendor changed to Autel
> 4:39:06 PM - 9 minutes ago
> 
> charger.Model changed to MaxiChargerAC
> 4:39:06 PM - 9 minutes ago
> 
> charger.ID changed to REDACTED
> 4:39:06 PM - 9 minutes ago
> 
> charger.Id.Tag changed to 01AUTEL0000000000001
> 4:39:06 PM - 9 minutes ago
> 
> charger.Reconnects changed to 298
> 4:39:06 PM - 9 minutes ago

**Comment by moshiko2312** [comment]
I have to this charger.
Please help us to configure.
I need i will pay for it.
Thanks

**Reply by jds11111** [reply]
It works for me. Set up the integration. Use the Autel app to navigate to your charger, and click on the gearbox in the upper left corner. Look for OCPP server (you might have to reopen to see it). Click on that and fill in the details for your HA ip and whatever port you set up in the integration. Your phone has to be connected to the charger by BLE to work. It all just worked for me (not using https), and gave charger on/off, availability and current control. There were also 10 sensors.  There are more details [here](https://docs.evcc.io/docs/devices/chargers#autel-ac-maxicharger-ac-compact-ocpp).

**Reply by nickdos** [reply]
You've linked to the EVCC project - completely unrelated to this integration.

I'm also not able to charge my MG4 with this integration - aviailability sensor changes from "on" to "off" as soon as I plug my charger into the car. I also can't change the max rate, I get an error as reported in issue #1019. 

EVCC works, so I'll go with that until #1019 is fixed or Autel release an updated firmware with better OCPP support.

**Comment by moshiko2312** [comment]
Hi, thank you very much, it works :-)

Is it possible that ocpp will not send notifications to ha?

**Reply by jds11111** [reply]
I don't follow you. I get three controls and ten sensors updated.
